code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1612,3 +1612,32 @@
 
 	msg += get_display_species()
 	return msg
+
+/mob/living/carbon/human/pull_damage()
+	if(((health - halloss) <= config.health_threshold_softcrit))
+		for(var/name in organs_by_name)
+			var/obj/item/organ/external/e = organs_by_name[name]
+			if(!e)
+				continue
+			if((e.status & ORGAN_BROKEN && (!e.splinted || (e.splinted && e.splinted in e.contents && prob(30))) || e.status & ORGAN_BLEEDING) && (getBruteLoss() + getFireLoss() >= 100))

```

Comment:
In `(!e.splinted || (e.splinted && e.splinted in e.contents && prob(30)))`, the second `e.splinted` (first within the second case) is already enforced by the `!e.splinted` in the first case. That is, the second case will only be evaluated if the first is false, which is to say, if `e.splinted != NULL`.

Reply:
Not my code, but yes, true.

Reply:
Changed